mycroft/skills/phillips_hue/vocab/en-us/DecreaseColorTemperatureKeyword.voc

```diff
@@ -0,0 +1,10 @@
+warm my lights
+warm the lights
+decrease the color temperature of my lights
+decrease the color temperature of the lights
+decreate the temperature of my lights
+decreate the temperature of the lights
+turn down the color temperature of my lights
+turn down the color temperature of the lights
+turn down the temperature of my lights
+turn down the temperature of the lights
```

Comment:
Similar what I suggested before, use a `LightKeyword` and `DecreaseColorTemperatureKeyword` here to clean up the vocab. The [Volume skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/volume) has good examples.